aten/src/ATen/native/cpu/DistributionTemplates.h

```diff
@@ -180,9 +180,7 @@ void normal_kernel(Tensor& self, double mean, double std, RNG generator) {
     normal_fill(self, static_cast<float>(mean), static_cast<float>(std), generator);
 #endif
   } else {
-    // bfloat16 cannot be properly tested due to the lack of other operations
-    // like add/sub/mean implemented for half
-    AT_DISPATCH_FLOATING_TYPES_AND_HALF(self.scalar_type(), "normal_kernel_cpu", [&] {
+    AT_DISPATCH_FLOATING_TYPES_AND2(kHalf, kBFloat16, self.scalar_type(), "normal_kernel_cpu", [&] {
```

Comment:
Thanks for fixing this. I was trying to create a bfloat tensor on some of my PR and this block me from doing so.

Reply:
is this related to this change?  Test?

Reply:
make_tensor already has a solution for this right?  Using that and splitting up the PRs seems better.

Reply:
Moved this change  to https://github.com/pytorch/pytorch/pull/47143